app/src/main/java/org/simple/clinic/di/DateFormatterModule.kt

```diff
@@ -28,4 +28,9 @@ class DateFormatterModule {
   @AppScope
   @Named("recent_patients_header")
   fun providesTimeFormatterForRecentPatientsHeader(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM, yyyy", locale)
+
+  @Provides
+  @AppScope
+  @Named("bp_entry")
+  fun providesTimeFormatterForBpEntry(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM, yyyy", locale)
```

Comment:
This looks like a utility module with a global scope.
I am not entirely sure that using this as a global module is of any use since even though the patterns are same for `recent_patients_header` and `bp_entry`, we are still creating a new pattern based on the context where it is used.
We much rather scope them to the classes where it is used than use app scope.
Wdyt?